frontend/src/auth/AuthService.jsx

```diff
@@ -45,7 +45,7 @@ export default class AuthService {
   signinRedirectCallback = () => {
     this.UserManager.signinRedirectCallback()
       .then(() => {
-        '';
+        console.log('signin redirect callback executed');
```

Comment:
`// eslint-disable-line no-console`

Reply:
complete file is /*eslint-disable */, lots of console.log...